apps/rotm/translations/src/en/email.json

```diff
@@ -0,0 +1,19 @@
+{
+  "caseworker": {
+    "subject": "Report on Terrorism Material: new request",
+    "fields": {
+      "uniqueId": {
+        "label": "Application ID"
```

Comment:
Shouldn't this be `Id`?

Reply:
I wouldn't have thought so, but I'll check the ticket.